.circleci/config.yml

```diff
@@ -222,26 +226,51 @@ commands:
 jobs:
   api-build-test:
     <<: *java_defaults
+    parallelism: 3
     steps:
       - checkout_init_git
       - ensure_branch_has_changes:
           dir_name: "api"
       - manage_api_cache:
           restore: true
       - run:
-          name: Validate Swagger definitions
+          name: Run Java unit tests
           working_directory: ~/workbench/api
-          command: ./project.rb validate-swagger --project-prop verboseTestLogging=yes
+          # Refers to https://circleci.com/docs/2.0/language-java/#sample-configuration
```

Comment:
Can you add more to this comment? The shell script is pretty hairy and it's not clear what it's actually doing. Presumably it has something to do with parallelizing the tests